server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/model/message/view/MessageMetadataViewFactoryTest.java

```diff
@@ -40,6 +40,7 @@
 import org.apache.james.mailbox.model.MailboxPath;
 import org.apache.james.mailbox.model.MessageRange;
 import org.apache.james.mailbox.model.MessageResult;
+import org.apache.james.util.ClassLoaderUtils;
```

Comment:
Please add the same tests you did in MessageFullViewFactory regarding `hasAttachmentShouldReturnFalseWhenAllAttachmentsAreInline`.

I bet we do have an issue.